From 9e1171cdf76e9cb5169a32254669e7332751989e Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Fri, 16 Sep 2022 02:17:08 +0200
Subject: [PATCH 1/2] add clang/llvm to pyarrow{,-tests} host dependencies

LLVM headers needed by gandiva; clang needed because
FindLLVMAlt.cmake depends on it to find LLVM correctly.

Co-Authored-By: Sutou Kouhei <kou@clear-code.com>
---
 recipe/meta.yaml | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/recipe/meta.yaml b/recipe/meta.yaml
index fd0625296..4e3cce3ba 100644
--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -200,10 +200,12 @@ outputs:
         - {{ compiler("cuda") }}  # [cuda_compiler_version != "None"]
       host:
         - {{ pin_subpackage('arrow-cpp', exact=True) }}
+        - clangdev {{ llvm_version }}
+        - llvmdev {{ llvm_version }}
         - cython
         - numpy
-        - python
         - openssl
+        - python
         - setuptools
         - setuptools_scm
         - six
@@ -278,10 +280,12 @@ outputs:
       host:
         - {{ pin_subpackage('arrow-cpp', exact=True) }}
         - {{ pin_subpackage('pyarrow', exact=True) }}
+        - clangdev {{ llvm_version }}
+        - llvmdev {{ llvm_version }}
         - cython
         - numpy
-        - python
         - openssl
+        - python
         - setuptools
         - setuptools_scm
         - six
-- 
2.37.0.windows.1

